bindings/cs/rl.net.cli.test/RegressionTest.cs

```diff
@@ -0,0 +1,94 @@
+/*
+
+
+        [TestMethod]
+        public void Test_PPTDesigner()
+        {
+            
+        }
+
+*/
+
```

Comment:
Should this be here?

Reply:
Nope, good catch. Also there was a superfluous reference to the corresponding test file in the project.